be/src/storage/olap_server.cpp

```diff
@@ -77,6 +77,9 @@ Status StorageEngine::start_bg_threads() {
     _update_cache_expire_thread = std::thread([this] { _update_cache_expire_thread_callback(nullptr); });
     Thread::set_thread_name(_update_cache_expire_thread, "cache_expire");
 
+    _update_cache_evict_thread = std::thread([this] { _update_cache_evict_thread_callback(nullptr); });
```

Comment:
Which thread poll incharge of the update cache eviction before?

Reply:
> Which thread poll incharge of the update cache eviction before?

We don't evict update cache before. We only remove the entry from cache when the entry is expired and the thread is `_update_cache_expire_thread`.